modules/decision_reviews/lib/decision_reviews/notification_callbacks/form_notification_callback.rb

```diff
@@ -5,10 +5,12 @@
 
 module DecisionReviews
   class FormNotificationCallback < ::VeteranFacingServices::NotificationCallback::SavedClaim
+    attr_reader :reference
```

Comment:
[::VeteranFacingServices::NotificationCallback::SavedClaim](https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/veteran_facing_services/notification_callback/saved_claim.rb#L12) inherits from [::VeteranFacingServices::NotificationCallback::Default](https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/veteran_facing_services/notification_callback.rb#L22), where the initialize method iterates through the `metadata` keys and [sets each as an instance variable](https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/veteran_facing_services/notification_callback.rb#L72), which allows us to access `reference` here.